drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```diff
@@ -198,6 +198,10 @@ config NRF70_UTIL
 	depends on SHELL
 	bool "Utility shell in nRF70 driver"
 
+config NRF70_DEBUG
```

Comment:
Perhaps `NRF70_DEBUG_SHELL` instead to make it clear that this is about the shell?